packages/runtime/container-runtime/src/dataStoreContext.ts

```diff
@@ -260,6 +260,15 @@ export abstract class FluidDataStoreContext
 	private readonly thresholdOpsCounter: ThresholdCounter;
 	private static readonly pendingOpsCountThreshold = 1000;
 
+	/**
+	 * If the summarizer makes local changes, a telemetry event is logged. This has the potential to be very noisy.
+	 * So, adding a threshold of how many telemetry events can be logged per data store context. This can be
+	 * controlled via feature flags.
+	 */
+	private readonly localChangesTelemetryThreshold?: number;
+	// The count of the local changes in summarizer telemetry that has been logged by this data store context.
+	private localChangesTelemetryCount = 0;
```

Comment:
You can model same state with one variable - a counter that decrements from max to zero

Reply:
Thanks for the suggestion. Changed it to use one variable.